pkg/cmd/server/start/start_master.go

```diff
@@ -356,7 +356,7 @@ func (o MasterOptions) CreateCerts() error {
 
 func BuildKubernetesMasterConfig(openshiftConfig *origin.MasterConfig) (*kubernetes.MasterConfig, error) {
 	if openshiftConfig.Options.KubernetesMasterConfig == nil {
-		return nil, nil
+		return nil, fmt.Errorf("external Kubernetes mode is not supported anymore")
```

Comment:
`KubernetesMasterConfig is required to start this server - using an external Kubernetes is no longer supported`.